docs/platform/what-is-tizen/versions/tizen-6-0-m2.md

```diff
@@ -0,0 +1,101 @@
+
+The following is updated in Tizen 6.0:
+
+- **.NET Core**:
+     - Latest .NET Core 3.1(LTS)
+     - Optimized performance
+     - Diagnostics tools
+- **TizenFX**:
+    - Tizen platform-specific API set
+    - API level 8 released
+- **Xamarin**:
+    -  Latest Xamarin.Forms (4.8.0)  and Xamarin.Essentials (1.5.3) are available.
+    - TV UI Controls and CircularUI: New UI controls for Smart TVs and Smart Watches.
+    - XAML Hot Reload: Boosting developer productivity
+
+### Web framework
+
+Tizen 6.0 provides a new user experience through device-to-device connectivity and interworking.
+
+- Web-based device offloading services
+
+As the complexity of applications in the web environment increases, a method of maximizing computational power is needed.
+
+Tizen 6.0 provides **multi-device offloading service** technology using multiple devices:
+
+![img](media/6.0_1_web.png)
+
+### Lightweight Web Solution
+
+Escargot is the Tizen JavaScript engine for the product having hardware resource restriction. It uses about 14% in binary size and 35% in memory usage for the benchmark application compared to those of Chrome's V8 JavaScript engine. Escargot supports ES6 full specification and also supports ES 10 up to a large extent:
+
+ The following are the details for memory size and features in comparison to Chrome V8 benchmarks:
+
+  - Memory size  (vs chrome v8) octane benchmark in Ubuntu 18.04
+    - Binary size : 13.7 %
+    - Average RSS : 35.1 %
+
+  - Features
+    - Support ES6 Full feature and partial support of ES10
+
+## Machine Learning framework
+
+NNSteamer is an on-device AI framework that makes it easy to configure complex artificial neural networks in the application by making artificial neural network configurations with Gstreamer's plugin. 	 
+
+In Tizen 6.0, NN Runtime supports control-flow and dynamic tensor that enables voice model acceleration. In addition, by providing the NN Compiler front-end for NN Runtime, it can convert well-known models such as Tensorflow and TensorFlow-lite to be used on CE devices with insufficient resources on NN Runtime.	 
+
+- NNStreamer 	 
+ - Neural Network Pipeline. 	 
+   - .NET and C APIs supported. 	 
+   - Various accelerating hardware and frameworks supported. 	 	 
+   - Implement AI app more easily. 	 
+   - Execute AI app more efficiently. 	 
+
+- NNTrainer 	 
+ - On-Device Training. 	 
+   - Transfer learning, meta learning, and reinforcement learning. 	 
+   - APIs to construct and update neural network layers.
+
+- NN Runtime 	 
+ - Tizen’s default Neural Network inference framework (since v5.0).
+ - CPU and GPU mixed acceleration with profiling based optimal scheduling (since v5.5).
+ - Useful features for speech model support (in v6.0). 	 
```

Comment:
Yes it was rendering properly, but now Git is not rendering images. 

But can you change the list indent: 
```suggestion
- NN Runtime 	 
    - Tizen’s default Neural Network inference framework (since v5.0).
    - CPU and GPU mixed acceleration with profiling based optimal scheduling (since v5.5).
    - Useful features for speech model support (in v6.0). 	 
    - Control-flow, Dynamic Tensor, Shape Inference, etc.

Reply:
If you wish to keep the current indentation. The only thing that we were concerned about was the sequence of the previous bullets. Like: 
 - text
 - text 
     - text
     - text 
- text 
- text 
    - text 
    - text
- NNStreamer

- Tizen’s default Neural Network inference framework (since v5.0).
- CPU and GPU mixed acceleration with profiling based optimal scheduling (since v5.5).
- Useful features for speech model support (in v6.0). 	  
    - Control-flow, Dynamic Tensor, Shape Inference, etc       

So this list indent is intentional, then it is fine.

Reply:
The list indent is intentional. so it's fine. Thanks~

Reply:
> The list indent is intentional. so it's fine. Thanks~

In markdown page, to give indentation more than two spaces are needed.
However in this page, only one space is used to give indentation so some indentation are ignored.

The layout of the indentation is different between the raw md file and rendered page,
and that's the reason why reviewers are asking to check again.

I think you'd better check https://docs1.stg.tizen.org/staging/1451/platform/what-is-tizen/versions/tizen-6-0-m2/.
The indentation to second level are all removed.

Reply:
I got it. When I test it in mark up viewer, it seems ok but I come to know there is the issue in web site.

I changed as follows ( second level..)

- NNStreamer ( Neural Network Pipeline ) 	 
  - .NET and C APIs supported. 	 
  - Various accelerating hardware and frameworks supported. 	 	 
  - Implement AI app more easily. 	 
  - Execute AI app more efficiently. 	 

- NNTrainer ( On-Device Training )
  - Transfer learning, meta learning, and reinforcement learning. 	 
  - APIs to construct and update neural network layers.

- NN Runtime 	 
 - Tizen’s default Neural Network inference framework (since v5.0).
 - CPU and GPU mixed acceleration with profiling based optimal scheduling (since v5.5).
 - Useful features such as Control-flow, Dynamic Tensor, Shape Inference, etc. for speech model support

- NN Compiler in Tizen SDK 	 
 - Support well known ML frameworks, and model formats such as Tensorflow, Tensorflow-lite (preferential), Caffe, ONNX (experimental).